build-system/server/app-utils.js

```diff
@@ -127,28 +127,28 @@ const replaceUrls = (mode, file, hostName, inabox, storyV1) => {
     }
   } else if (mode == 'compiled') {
     file = file.replace(
-      /https:\/\/cdn\.ampproject\.org\/v0\.(m?js)/g,
-      hostName + '/dist/v0.$1'
+      /https:\/\/cdn\.ampproject\.org\/v0\.(m|sxg\.)?js/g,
+      hostName + '/dist/v0.$1js'
```

Comment:
hmm this kind reverts back the recommendation that @rcebulko had given me in that the capture group should also have the js so that the replace is simpler

Reply:
That's true. I'll update it accordingly.